.github/workflows/test.yml

```
@@ -157,10 +157,14 @@ jobs:
       - name: Install Postgres, setup DB and auth, and wait for server start
         run: |
           export PATH="$(brew --prefix)/opt/${POSTGRES_FORMULA}/bin:$PATH" PGDATA=/tmp/vapor-postgres-test
-          (brew unlink postgresql || true) && brew install "${POSTGRES_FORMULA}" && brew link --force "${POSTGRES_FORMULA}"
+          # ** BEGIN ** Work around bug in both Homebrew and GHA
+          (brew upgrade python@3.12 || true) && (brew link --force --overwrite python@3.12 || true) && brew upgrade
+          # ** END ** Work around bug in both Homebrew and GHA
+          brew install --overwrite "${POSTGRES_FORMULA}"
+          brew link --overwrite --force "${POSTGRES_FORMULA}"
```

Let's revert those changes for now
```suggestion
          (brew unlink postgresql || true) && brew install "${POSTGRES_FORMULA}" && brew link --force "${POSTGRES_FORMULA}"
```